src/components/pacbio/PacbioLibraryEdit.vue

```diff
@@ -94,4 +100,14 @@ const updateLibrary = async () => {
     showAlert(`Failed to update library in Traction: ${errors}`, 'danger')
   }
 }
+
+const validateFields = (library) => {
+  const requiredAttributes = ['id', 'template_prep_kit_box_barcode', 'volume', 'concentration']
+  const errors = requiredAttributes.filter((field) => !library[field])
+
+  return {
+    success: errors.length === 0,
+    errors,
+  }
+}
```

Comment:
Would it be better to keep this validation in store? Also, I think the errors need to be populated against the field objects to make <traction-field-error> to work.